modules/ROOT/partials/configuration/format_noneditable_selector.adoc

```diff
@@ -0,0 +1,20 @@
+[[format_noneditable_selector]]
+== `+format_noneditable_selector+`
```

Comment:
Do you need to mention anything about `data-mce-cef-wrappable` here? As that seems important to know it's actually part of the defaults that cannot be changed and applies automatically to elements with that attribute.

Reply:
Yeah, I wasn't sure whether to mention it or not. I guess since it can't be overridden in any way by the integrator, then having a note about it would be good.